.travis.yml

```diff
@@ -57,6 +61,7 @@ deploy:
   # deploy nightly build to github release
   - provider: releases
     api_key: $GIT_TOKEN
+    prerelease: true

```

Comment:
>prerelease: true [](start = 4, length = 16)

Keep consistency to use the $IS_TEST variable if it is used elsewhere

Reply:
For the "deploy nightly build to github release" deployment, `prerelease` tag will always be true.
For the "deploy to github release" deployment, it can be production release or RC version release (which will have `$IS_TEST == true`). `prerelease` tag will only be true when $IS_TEST is true.